tests/dark_storage/conftest.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+import os
+import shutil
+import py
+
+from starlette.testclient import TestClient
+from ert_shared.dark_storage import enkf
+
+
+@pytest.fixture
+def poly_example_tmp_dir(tmpdir, source_root, monkeypatch):
+    monkeypatch.setenv("ERT_DARK_STORAGE_CONFIG", "poly.ert")
+    poly_dir = py.path.local(os.path.join(str(tmpdir), "poly_example"))
+    shutil.copytree(
+        os.path.join(source_root, "test-data", "local", "poly_example"),
+        poly_dir,
+    )
+
+    with poly_dir.as_cwd():
+        yield
+
+
+@pytest.fixture
+def dark_storage_client(dark_storage_app):
+    return TestClient(dark_storage_app)
+
+
+@pytest.fixture
+def env(monkeypatch):
+    monkeypatch.setenv("ERT_STORAGE_DATABASE_URL", "sqlite://")
```

Comment:
I don't see why we need the database_url set. We are running without _--enable-new-storage_ right? At least we should. Just running the test in ert-storage also creates a _storage_  folder in the runpath. I don't understand why this should be like that :thinking:

Reply:
I fixed the storage folder creation at least

Reply:
yes, i can see the url is needed due to us importing ert-storage to verify compability between the two api's. Created an issue on this.